kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -554,6 +554,35 @@ public void testGetCaseInstancesWithData() {
         assertEquals("first case started", instance.getCaseFile().getData().get("s"));
     }
 
+    @Test
+    public void testByPassAuthCaseTasksAssignedAsPotentialOwner() throws Exception {
+        String caseId = startUserTaskCase(USER_YODA, USER_JOHN);
+        assertNotNull(caseId);
+        assertTrue(caseId.startsWith(CASE_HR_ID_PREFIX));
+        try {
+            List<TaskSummary> tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_YODA, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(1, tasks.size());
+            assertEquals("Hello1", tasks.get(0).getName());
```

Comment:
Before changing to user Johh, I would also check here first that the number of user John are 0. This will be required to confirm that the number of tasks retrieved later on when sending an empty user are still being 0.